pysatMadrigal/instruments/methods/general.py

```diff
@@ -34,6 +39,451 @@ def cedar_rules():
+        5600: 'jbs_{{year:04d}}{{month:02d}}{{day:02d}}.{{version:03d}}.',
+        5700: 'spm{{year:02d}}{{month:02d}}{{day:02d}}g.{kindat}.',
+        5720: 'dbm{{year:02d}}{{month:02d}}{{day:02d}}g.{kindat}.',
+        5860: 'stm{{year:02d}}{{month:02d}}{{day:02d}}g.{kindat}.',
+        5900: 'sfm{{year:02d}}{{month:02d}}{{day:02d}}g.{kindat}.',
+        5950: 'rbm{{year:02d}}{{month:02d}}{{day:02d}}g.{kindat}.',
+        5980: 'eum{{year:02d}}{{month:02d}}{{day:02d}}g.{kindat}.',
+        6205: 'akl{{year:02d}}{{month:02d}}{{day:02d}}g.*.',
+        6206: 'asl{{year:04d}}{{month:02d}}{{day:02d}}.{{version:03d}}.',
+        6300: 'uil{{year:02d}}{{month:02d}}{{day:02d}}g.{kindat}.',
+        6320: 'Colorado State sodium lidar',
+        6330: 'usl{{year:02d}}{{month:02d}}{{day:02d}}g.{kindat}.',
+        6340: 'alo*.',
+        6350: 'nlo*.',
+        6360: 'cul*.',
+        6370: 'unl*.',
+        6380: 'pfl{{year:04d}}{{month:02d}}{{day:02d}}_{{cycle:03d}}.',
+        7190: 'usi*.',
+        7192: 'amp{{year:02d}}{{month:02d}}{{day:02d}}?.{{version:03d}}.',
+        7200: 'mhi{{year:04d}}{{month:02d}}{{day:02d}}.{kindat}.',
+        7201: 'aai{{year:04d}}{{month:02d}}{{day:02d}}.{kindat}.',
+        7202: 'abi{{year:04d}}{{month:02d}}{{day:02d}}.{kindat}.',
+        7203: 'eai{{year:04d}}{{month:02d}}{{day:02d}}.{kindat}.',
+        7204: 'mai{{year:04d}}{{month:02d}}{{day:02d}}.{kindat}.',
+        7205: 'rai{{year:04d}}{{month:02d}}{{day:02d}}.{kindat}.',
+        7206: 'jci{{year:04d}}{{month:02d}}{{day:02d}}.{kindat}.',
+        7240: 'mhi*.',
+        7580: 'qac*.',
+        11: 'j??*{{year:02d}}{{month:02d}}{{day:02d}}g.{{version:03d}}.',
+        840: 'jul{{year:04d}}{{month:02d}}{{day:02d}}_esf.{{version:03d}}.',
+        3000: 'utx*.',
+        8001: '????_?_??.gps_all.out.',
+        8000: '*{{year:02d}}{{month:02d}}{{day:02d}}*.{{version:03d}}.',
+        8002: '????_?_??.gps_all.out.',
+        8010: 'scin_{{year:04d}}{{month:02d}}{{day:02d}}.{{version:03d}}.',
+        3010: 'dvs{{year:02d}}{{month:02d}}{{day:02d}}g.{kindat}.',
+        3320: 'wup{{year:02d}}{{month:02d}}{{day:02d}}g.{kindat}.',
+        4470: 'p4p{{year:02d}}{{month:02d}}{{day:02d}}g.{kindat}.',
+        4473: 'y4p{{year:02d}}{{month:02d}}{{day:02d}}g.{kindat}.',
+        4480: 'arp{{year:04d}}{{month:02d}}{{day:02d}}.{{version:03d}}.',
+        4481: 'agp{{year:04d}}{{month:02d}}{{day:02d}}.{{version:03d}}.',
+        7191: 'mtm{{year:02d}}{{month:02d}}{{day:02d}}g.{kindat}.'}
+
+    # Warn if file format not available
+    msg = ""
+    if inst_code not in format_str.keys():
+        msg = "".join(["file format string not available for ",
+                       "instrument code {:d}: ".format(inst_code)])
+        fstr = "*."
+    else:
+        fstr = format_str[inst_code]
+
+    # Warn if file format has multiple '*' wildcards
+    num_wc = len(fstr.split("*"))
+    if num_wc >= 3:
+        msg = "".join(["file format string has multiple '*' ",
```

Comment:
Also got this,
```
In [1]: import pysat
import 
In [2]: import pysatMadrigal as py_mad
pysat WARNING: file format string has multiple '*' wildcards, may not be parsable by pysat
pysat WARNING: file format string has '*' between formatting constraints, may not be parsable by pysat
```

Reply:
Now that one is confusing.  No idea why that one appears on module import.

Reply:
So, this comes from the Jicamarca init.  I think I can fix this by adding a "verbose" flag to the function.

Reply:
This should be fixed now.